embulk-parser-csv/build.gradle

```diff
@@ -62,6 +71,96 @@ javadocJar {
     from rootProject.file("LICENSE")
 }
 
+publishing {
+    publications {
+         maven(MavenPublication) {
+            afterEvaluate { project ->
+                groupId = "${project.group}"
+                artifactId = "${project.name}"
+            }
+
+            from components.java
+             // javadocJar and sourcesJar are added by java.withJavadocJar() and java.withSourcesJar() above.
+             // See: https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html
+
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                packaging "jar"
+
+                afterEvaluate { project ->
+                    name = "${project.name}"
+                    description = "${project.description}"
+                }
+                url = "https://www.embulk.org/"
+
+                licenses {
+                    license {
+                        // http://central.sonatype.org/pages/requirements.html#license-information
+                        name = "The Apache License, Version 2.0"
+                        url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+
+                developers {
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Shinichi Ishimura"
+                        email = "shiketaudonko41@gmail.com"
+                    }
+                    developer {
+                        name = "Naotoshi Seo"
+                        email = "sonots@gmail.com"
+                    }
```

Comment:
@sonots We're going to separate `embulk-standards` plugins as individual plugins, then to release them as independent pure-Java Embulk plugin JARs into Maven Central. In this `embulk-parser-csv`, we wanted to include you as a developer from the pull requests #434 and #440. What do you think?

Reply:
@dmikurube I do not care either including me or not, but if this means a maintainer, probably I’m not the one.

Reply:
"Maintainers" would be listed out of this parts (now in GitHub Teams). I intended this list would be just "developers who contributed a meaningful code change / an additional feature". (Just typo fixes and cosmetic changes are not included.)